third_party/move/move-vm/runtime/src/storage/module_storage.rs

```diff
@@ -57,12 +57,27 @@ pub trait ModuleStorage: WithRuntimeEnvironment {
 
     /// Returns the size of a module in bytes, or [None] otherwise. An error is returned if the
     /// there is a storage error.
-    fn fetch_module_size_in_bytes(
+    ///
+    /// Note: this API is not metered!
+    fn unmetered_get_module_size(
         &self,
         address: &AccountAddress,
         module_name: &IdentStr,
     ) -> VMResult<Option<usize>>;
 
+    /// Returns the size of a module in bytes if it exists. An error is returned if the module does
+    /// not exist, or if there is a storage error.
+    ///
+    /// Note: this API is not metered!
+    fn unmetered_get_existing_module_size(
```

Comment:
I'd make it a trait method if we had >1 uses, but it's okay if you want to keep it.

Reply:
Yes, removed, moved implementation to that 1 usage